lib/msal-browser/src/app/ClientApplication.ts

```diff
@@ -69,7 +69,7 @@ export abstract class ClientApplication {
      * In Azure B2C, authority is of the form https://{instance}/tfp/{tenant}/{policyName}/
      * Full B2C functionality will be available in this library in future versions.
      *
-     * @param {@link (Configuration:type)} configuration object for the MSAL PublicClientApplication instance
+     * @param configuration Object for the MSAL PublicClientApplication instance
```

Comment:
tsdoc  tip: format apparently is supposed to be @param `parameter` - `short description`